azext_iot/common/certops.py

```diff
@@ -125,3 +134,198 @@ def open_certificate(certificate_path: str) -> str:
         raise ValueError("Certificate file type must be either '.pem' or '.cer'.")
     # Remove trailing white space from the certificate content
     return certificate.rstrip()
+
+
+# TODO - Unit test, compare with test_utils::_generate_root_certificate
+def create_root_certificate(
+    subject: Optional[str] = "Azure_IoT_CLI_Extension_Cert",
```

Comment:
A subject str is always required right? We should only use `Optional` if `None` is allowed.